packages/graphql/src/schema-model/parser/annotations-parser/authentication-annotation.ts

```diff
@@ -33,7 +32,7 @@ const authenticationDefaultOperations: AuthenticationOperation[] = [
     "DELETE_RELATIONSHIP",
     "SUBSCRIBE",
 ];
-export function parseAuthenticationAnnotation(directive: DirectiveNode): AuthenticationAnnotation {
+export const parseAuthenticationAnnotation: AnnotationFactory<AuthenticationAnnotation> = (directive) => {
```

Comment:
For consistency (and to avoid some gotcha) I'd recommend keeping this as a `function` for top level named functions.

I guess you made this change to add the `AnnotationFactory` types, but because the `annotationsParsers` are now statically set in the array with the type set, this shouldn't be needed (the type will fail if the function does not match)